packages/itinerary-body/src/stories/itinerary-body-defaults-wrapper.tsx

```diff
@@ -87,6 +89,9 @@ export default class ItineraryBodyDefaultsWrapper extends Component<
         mapillaryKey="fake key, but ok because the api response is also fake"
         PlaceName={PlaceName || DefaultPlaceName}
         RouteDescription={RouteDescription || DefaultRouteDescription}
+        RouteDescriptionFooter={
+          RouteDescriptionFooter || DefaultRouteDescriptionFooter
```

Comment:
The default should be `undefined`

Reply:
If I make it undefined, then it never shows up in any stories at all.

Reply:
Can you define it in the story?

Reply:
I can try. I'm unfamiliar with doing so, but I'll give it a whirl.

Reply:
Oh, my bad, I think I misunderstood what you asked. I think. I made the following change and things seem to be right as rain again.

![image](https://github.com/opentripplanner/otp-ui/assets/31577015/e001b453-424f-488f-9404-83c47ed99aad)